Source/WebCore/testing/Internals.idl

```diff
@@ -1159,4 +1159,5 @@ typedef (FetchRequest or FetchResponse) FetchObject;
 
     [Conditional=ARKIT_INLINE_PREVIEW_MAC] Promise<sequence<DOMString>> modelInlinePreviewUUIDs();
     [Conditional=ARKIT_INLINE_PREVIEW_MAC] DOMString modelInlinePreviewUUIDForModelElement(HTMLModelElement modelElement);
+    undefined avoidIOSurfaceSizeCheckInWebProcess(Element element);
```

Comment:
Is there a reason this API is not passed an `HTMLCanvasElement`?

Reply:
This was originally due to a linker error. Adding ExportMacro=WEBCORE_EXPORT to HTMLCanvasElement fixed this issue.